.codeclimate.yml

```diff
@@ -29,7 +29,7 @@ plugins:
         enabled: false
   eslint:
     enabled: true
-    channel: "eslint-7"
+    channel: "eslint-8"
```

Comment:
This fixes CodeClimate and allows it to check the repo - once it's merged to master we'll be able to do a scan of the repo again.

![image](https://github.com/bcgov/common-hosted-form-service/assets/35532993/7a32a0c2-8dd6-460e-a040-9c7bc3dd1449)

Reply:
https://codeclimate.com/github/bcgov/common-hosted-form-service/builds